kits/real-estate-analyst/apps/.env.example

```
@@ -0,0 +1,3 @@
+LAMATIC_CONFIG_REALESTATE=
+PROPERTY_SHEET_ID=
+GOOGLE_SHEETS_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_SHEETS_API_KEY key should go before the LAMATIC_CONFIG_REALESTATE key

(UnorderedKey)